man/man1/exiv2.1

```diff
@@ -211,7 +211,7 @@ or 'm'(ute). The default log-level is 'w'. \fB\-Qm\fP is equivalent
 to \fB\-q\fP. All log messages are written to standard error.
 .TP
 .B \-b
-Show large binary values (default is to suppress them).
+The output from large binary values is to restricted by default.  When outputting the value of a large value such as MakerNote, the default behaviour is to restrict the data and end with "...".  The option --binary enables you to see all the data.
```

Comment:
Thanks for the explanation!

Reply:
I never knew what that did either.  It's now useful.

If we every really want to provide an option to output aw/binary data, we can use the option --raw/-R.  It's a pity that Andreas called that --binary/-b, you should think of it as "bloody long output!".